scikits/fitting/tests/test_linlstsq.py

```diff
@@ -76,7 +76,7 @@ def test_vs_numpy(self):
         """LinearLeastSquaresFit: Compare fitter to np.linalg.lstsq."""
         interp = LinearLeastSquaresFit()
         interp.fit(self.x, self.y)
-        params = np.linalg.lstsq(self.x.T, self.y)[0]
+        params = np.linalg.lstsq(self.x.T, self.y, rcond=None)[0]
```

Comment:
This unfortunately gives an error (`TypeError: a float is required`) in numpy < 0.14. So if you do this, you'll need to specify a minimum of 0.14 in setup.py.

Reply:
I'll do `rcond=-1` instead, which should be backwards compatible. This only matters for one test anyway.